packages/notary-service-client/README.md

```diff
@@ -4,52 +4,6 @@
 
 ## Example
 
-## CLI
```

Comment:
The notary service is going to be decommissioned in the upcoming months, didn't see a good reason to keep this here